README.md

```diff
@@ -9,10 +9,7 @@ See [architecture overview](docs/README.md).
 You can simulate the core of dialog processing on the command line — by feeding the dialog engine with command-line entries rather than entries from a kinesis stream. Try it out by running `python simulator.py`.
 
 ## Deployment
-We deploy from the command line using the serverless framework. Once you are a member of the serverless project you can deploy to  
-
-- dev: `serverless deploy -s dev --env development`
-- prod: `serverless deploy -s prod --env production`
+Deployments are done via AWS CodeBuild. Log in to the CodeBuild UI to deploy dev and prod.
```

Comment:
Does the integration test run as part of the build?

Reply:
It does not. As of now the onus is on the developer to make sure that dev works before deploying to prod. We should be able to turn on dev -> integration test -> prod in code pipeline pretty easily, though